### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by gdarko** [InvoiceShelf/InvoiceShelf#16] [Announcements]
_What's new:_

* Upgraded to latest Laravel 10
* Upgraded to latest PHPUnit 10 / Pest 2
* Upgraded to latest Vue 3.4
* Upgraded to latest Vue-router, Vue-flatpickr and other JS libraries
* Upgraded to latest Vite 5.0
* Upgraded all PHP packages for PHP 8.2/8.3 compatibility
* Moved docker image in separate repository <a href="https://github.com/InvoiceShelf/docker" target="_blank">InvoiceShelf/docker</a>
* In-app updates will now pull from invoiceshelf.com
* Dropped unnecessary JS libraries and cleaned up JS code
* Dropped (git-ignored) the public/build directory. App builds can be downloaded from the website or GitHub releases page.
* Dropped Uffizzi (we can't support it right now)
* Improvements to the French translation (<a href="https://github.com/crater-invoice/crater/pull/1224" target="_blank">crater-invoice/crater@PR-1224</a>)
* Added support for Turkmen manat currency (<a href="https://github.com/crater-invoice/crater/pull/1133" target="_blank">crater-invoice/crater@PR-1133</a>)
* Added support for Bosnia and Herzegovina Convertible Mark currency (<a href="https://github.com/crater-invoice/crater/pull/1212" target="_blank">crater-invoice/crater@PR-1212</a>)
* Fixed hardcoded "?" in nginx rewrite rule (<a href="https://github.com/crater-invoice/crater/pull/819" target="_blank">crater-invoice/crater@PR-819</a>)
* Fixed payment note Amount field display (<a href="https://github.com/crater-invoice/crater/pull/1151" target="_blank">crater-invoice/crater@PR-1151</a>)
* Fixed duplicate character in generated hashes affecting PDFs (<a href="https://github.com/crater-invoice/crater/pull/1224" target="_blank">crater-invoice/crater@ISSUE-1150</a>)
* Fixed active menu item border not highlighted properly
* Fixed migrations rollback
* Changed PHP namespace from \Crater to \InvoiceShelf

To migrate from Crater, refer to the <a href="https://docs.invoiceshelf.com/migration-guide.html" target="_blank">Migration Guide</a>.

<hr /><em>This discussion was created from the release <a href='https://github.com/InvoiceShelf/InvoiceShelf/releases/tag/1.0.0'>v1.0.0</a>.</em>

**Comment by rihards-simanovics** [comment]
Hey @gdarko, are you able to join [this discord](https://discord.gg/5kCKBumdjM) server and so we could talk about combining forces and work on Crater fork together? Hope to see you there soon.
